_scicomputing/reference_training.md

```diff
@@ -57,20 +57,70 @@ Several groups on campus host weekly or monthly office hours to provide assistan
+  - [R Programming](https://www.coursera.org/specializations/data-science-foundations-r): A beginner-level program has five mini-courses. It takes about 4 months to complete.
+  - [Statistics with R](https://www.coursera.org/specializations/statistics): A beginner-level program with five mini-courses. It takes about 7 months to complete.
+  - [Genomic Data Science](https://www.coursera.org/specializations/genomic-data-science): An intermediate-level program for those who are already aquainted with R. It has eight mini-courses. It takes about 6 months to complete.
+  - [Python Programming](https://www.coursera.org/specializations/data-science-python): An intermediate-level program that takes about 4 months to complete.
+  - *Coursera offers a 7-day free trial, and is a paid subscription service after*
+
+- [Udacity](https://www.udacity.com): Offers courses on [Data Science](https://www.udacity.com/courses/school-of-data-science)
+  - *Udacity is a paid subscription service*
+  - [Currently offering one month free for their Nanodegree programs.](https://blog.udacity.com/2020/03/one-month-free-on-nanodegrees.html)
+
+- [Udemy](https://www.udemy.com): Offers courses on [Data Science](https://www.udemy.com/courses/development/data-science/)
+  - *Udemy offers courses at various price points.*
+  - Keep an eye out for sales which happen regularly and can drastically reduce the cost.
+
+- [CognitiveClass.ai](https://cognitiveclass.ai/) offers a wide range of courses focused on data science, AI, and cloud computing.
+  - *All courses are free*
+
+- [The Open Source Data Science Masters](http://datasciencemasters.org): An open-source curriculum for learning data science. This is a mixed media course made up of videos, books, and slides.
+  - *Some content is free, some is paid*
+  
 - [CalTech](http://work.caltech.edu/telecourse) Learning from Data
+  - *A free YouTube series*
+
+### Interactive Coding Platforms
+
+- [DataQuest](https://www.dataquest.io/): A subscription service that offers programs and courses focused on data anlysis and engineering in Python and R.
+    -*Teired payment system with basic and premium plans*
+- [CodeAcademy](https://www.codecademy.com/): A subscription service that offers coding programs and courses in many different languages.
+  -*Teired payment system with limited content available for free*
+  
+- [DataCamp](https://www.datacamp.com/): A subscription service that offers coding programs and courses in many different languages.
+  - We previously used DataCamp subscriptions in fredhutch.io courses, but changes in their licenses made this strategy cost prohibitive
+  - Anyone considering DataCamp services should be aware of [this incident involving their CEO](https://www.buzzfeednews.com/article/daveyalba/datacamp-sexual-harassment-metoo-tech-startup) before choosing to pay for their service.
+  -*Teired payment system with limited content available for free*
+
+### Free ebooks
```

Comment:
This issue comment includes a link to the Seattle public library and some resources available there. Please include here? 

https://github.com/FredHutch/wiki/issues/503#issue-583348173